Makefile

```diff
@@ -60,44 +63,54 @@ hiveadmission:
 run: generate fmt vet
 	go run ./cmd/manager/main.go --log-level=debug
 
+# Run against the configured Kubernetes cluster in ~/.kube/config
+.PHONY: run-operator
+run-operator: generate fmt vet
+	go run ./cmd/operator/main.go --log-level=debug
+
 # Install CRDs into a cluster
 .PHONY: install
-install: manifests
+install: crd rbac
 	kubectl apply -f config/crds
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 .PHONY: deploy
-deploy: manifests deploy-hiveadmission
-	kubectl apply -f manifests/
-	kubectl apply -f config/crds
+deploy: generate manifests
```

Comment:
it used to be that `make deploy` is all I needed to do to deploy Hive. Now it looks like we need to `make install` before to get the crd's in. Is that intentional? (Sorry for the trickle of comments, but I'm finally testing this :) )

Reply:
Actually make install can probably go away, the operator currently installs the CRDs. That may change, but make deploy should do the trick regardless.

Reply:
Right now it fails because the HiveConfig type is not defined.